src/shared/default-routes.js

```diff
@@ -17,7 +17,7 @@ export default ({
     allowEmptyResponse: true
   }
 }, {
-  path: '/:year/:monthnum/:day/:postname',
+  path: '/:category/:year/:monthnum/:postname',
```

Comment:
Presuming this change is for compatibility with `wordpress.org/news`, but the actual defaults for wordpress are as above. Could add a custom route into the bootstrapped project @thomasdigby ?